recipes-app/iot2050-firmware-update/iot2050-firmware-update_0.5.bb

```diff
@@ -11,15 +12,23 @@
 DESCRIPTION = "OSPI Firmware Update Scripts"
 MAINTAINER = "chao.zeng@siemens.com"
 
-SRC_URI = "file://iot2050-firmware-update.tmpl"
+SRC_URI = " \
+    file://update.conf.json.tmpl \
+    file://iot2050-firmware-update.tmpl"
 
-TEMPLATE_FILES = "iot2050-firmware-update.tmpl"
+TEMPLATE_FILES = "update.conf.json.tmpl iot2050-firmware-update.tmpl"
 
 inherit dpkg-raw
 
+DEBIAN_DEPENDS = "python3-progress"
+
 do_install() {
+    install -v -d ${D}/usr/share/iot2050/fwu
+    install -v -m 644 ${WORKDIR}/update.conf.json ${D}/usr/share/iot2050/fwu/
+    install -v -m 755 ${WORKDIR}/iot2050-firmware-update ${D}/usr/share/iot2050/fwu/
```

Comment:
The idea was to have to conf files in usr/share, not the scripts.

Reply:
Done, plz check